pkg/errors/errors.go

```diff
@@ -83,6 +83,9 @@ var (
 
 	// ErrDevPodDeleted raised if dev pod is deleted in the middle of the "okteto up" sequence
 	ErrDevPodDeleted = fmt.Errorf("development container has been removed")
+
+	//ErrDivertNotSupported raised if the divert feature is not supported in the current cluster
+	ErrDivertNotSupported = fmt.Errorf("the 'divert' field is only supported in namespaces managed by Okteto")
```

Comment:
can we add a hint to send the user to a page where she can learn more about this?

Reply:
yes, but we don't have it yet. I will add it when we have docs about divert